django-stubs/db/models/fields/related_descriptors.pyi

```diff
@@ -125,40 +125,41 @@ class ManyToManyDescriptor(ReverseManyToOneDescriptor, Generic[_To, _Through]):
     @property
     def through(self) -> type[_Through]: ...
     @cached_property
-    def related_manager_cls(self) -> type[ManyRelatedManager[_To]]: ...  # type: ignore[override]
+    def related_manager_cls(self) -> type[ManyRelatedManager[_To, _Through]]: ...  # type: ignore[override]
     @overload  # type: ignore[override]
     def __get__(self, instance: None, cls: Any = ...) -> Self: ...
     @overload
-    def __get__(self, instance: Model, cls: Any = ...) -> ManyRelatedManager[_To]: ...
+    def __get__(self, instance: Model, cls: Any = ...) -> ManyRelatedManager[_To, _Through]: ...
 
 # Fake class, Django defines 'ManyRelatedManager' inside a function body
 @type_check_only
-class ManyRelatedManager(Manager[_M], Generic[_M]):
+class ManyRelatedManager(Manager[_To], Generic[_To, _Through]):
```

Comment:
Just a thought: `ManyRelatedManager` gained a new type argument, but users may have usages that only contain 1 typevar.

Mypy 1.9 gained support for PEP 696 `TypeVar(default=...)` argument (using `typing_extensions.TypeVar`). We could make `_Through` default to `Model`, which would be a bit safer than mypy's implicit fallback `Any`.

Reply:
Sure, we can set a `default=`